DRAFT.md

```diff
@@ -1589,6 +1583,28 @@ such types are copy constructible. We make `polymorphic` unconditionally copy
 and move constructible. The authors do not envisage that this could be relaxed
 in a future version of the C++ standard.
 
+While a copy constructor cannot be a template, we can conditionally constrain
+copy construction of `indirect`, in C++20 and later, by defining:
+
+```c++
+indirect(const indirect& other) requires false = delete;
+
+template <typename TT=T>
+indirect(const indirect& other) requires is_copy_constructible_v<TT>;
+```
+
+An instantiation of the function template, with `TT=T`, is added to the overload
+set when `indirect` is copy-constructed and will be selected if the owned
+object type `T` is copy-constructible.
+
+This would make copy construction conditional for `indirect` but not for
+`polymorphic`. We opt for consistency and make copy construction unconditional
+for both `indirect` and `polymorphic`. Making `indirect` conditionally copy
+constructible in a future version of the C++ standard would require adding a
```

Comment:
copy-constructible

Reply:
I've gone for 'copy constructible' and de-hyphenated everywhere. US folk tend not to use hy-phens.